### DISCUSSION TITLE
Dupliquer un diagnostic d'une cantine à l'autre

### DISCUSSION THREAD

**Discussion by ghost** [betagouv/ma-cantine#1288] [Idées]
Encore une suggestion amélioration destinée aux gestionnaires de cuisine centrale.

**Problème :** aujourd'hui, si un gestionnaire de cuisine centrale souhaite remplir le diagnostic de chacune de ses cantines satellites, il est obligé de le faire à la main. Or quand les données qualitatives sont rigoureusement identiques d'un site à l'autre (ce qui est le cas par exemple pour les gestionnaires de la cuisine centrale d'Avignon, qui sert ~35-40 cantines satellites), cela représente une perte de temps pour le gestionnaire de remplir les N diagnostics.

**Solution proposée :** proposer une option "Dupliquer le diagnostic vers..." > "Cantine B / Cantine C / ..."

@JenniferStephan, @hfroot & @alemangui, votre avis?

**Comment by hfroot** [comment]
Quand il y a 35-40 diagnostics à remplir je pense que c'est un beau cas pour l'import massif. Est-ce que c'est une option pour cet utilisateur ?

**Reply by ghost** [reply]
Je ne pense pas que ce soit possible car l'import massif permet d'importer les données chiffrées, or c'est pour les éléments qualitatifs que les diags sont identiques d'une cantine à l'autre.

**Reply by Greta712** [reply]
j'ai également des cuisines centrales de taille plus petite < à 10 cuisines satellites qui m'ont demandé s'il était possible de dupliquer un diagnostic lorsque les infos sont identiques.

**Comment by ghost** [comment]
Besoin à nouveau exprimé par une utilisatrice (G S-L) le 28/06.

**Reply by hfroot** [reply]
Merci :) C'est maintenant possible dans la loi pour les cuisines centrales de télédéclarer pour tous leurs satellites avec un diagnostic. @valeriemerle ou @JenniferStephan  pourrait mieux éxpliquer comment ça marche
Pour un premier pas, on a maintenant l'option pour les centrales de créer et télédéclarer un diag, ainsi que l'option de lister tous les satellites (en staging). On va améliorer cette fonctionnalité au fur et à mesure.
Est-ce que tu pense qu'on est sûr le bon chemin de répondre à ces retours ?

**Reply by ghost** [reply]
Oui, je pense qu'on est sur la bonne voie! Merci. Cela répond au besoin exprimé pour la partie chiffrée, mais en revanche pas à celui exprimé pour la partie qualitative. 

Ex. : une diététicienne de Département qui gère des collèges, dont la partie qualitative du diagnostic est identique d'un établissement à l'autre car toutes les initiatives (sur la diversification des sources protéines, la lutte contre le gaspillage, la réduction de l'usage du plastique, et la communication aux convives) sont décidées à l'échelle du Département et ensuite mises en pratique dans tous les établissements.

**Reply by JenniferStephan** [reply]
hello ! je vois l'utilité... est-ce que le dpt est seul gestionnaire de tous les établissements ? est-ce qu'il y a qd même un gestionnaire au niveau de l'établissement ? pcq même si le dpt décide des démarches; est-ce qu'un gestionnaire sur place ne peut pas aussi en choisir d'autres ? et du coup remplir pr son établissement; uniquement la partie qualitative ?

**Reply by ghost** [reply]
Oui, dans ce cas précis, le Dpt est seul gestionnaire de tous les établissements.
Pour le coup, je ne sais pas si un gestionnaire sur site pourrait mettre en place d'autres démarches ; et si ce n'est pas le cas, pourquoi. Je suppose que le Dpt a décidé d'être seul gestionnaire pour épargner du temps de saisie des données aux gestionnaires sur site.

Dans d'autres cas, le gestionnaire de cuisine centrale aimerait remplir la partie qualitative uniquement de certaines cantines satellites que la CC livre. Mais le besoin reste le même : pouvoir dupliquer la partie qualitative du diag d'une cantine à l'autre.
